demo/test.html

```diff
@@ -138,7 +138,7 @@ <h3>Matrix</h3>
 
         let publishableKey = $('#publishable-key').val();
 
-        Radar.initialize(publishableKey);
+        Radar.initialize(publishableKey, { enableHighAccuracy });
```

Comment:
The other SDKs pass in `desiredAccuracy = high, medium, or low` to `trackOnce()` or set it on `trackingOptions` for `starTracking()`.   So for consistency we probably want to do the same thing for `trackOnce()` in the js sdk.  Since it's a boolean at the browser level, maybe we map `high` to `true` and `medium` and `low` to `false`.

thx @jsani-radar for flagging in slack

Reply:
Since 4.x is going to be a major version bump, that should give us cover to introduce `trackOnce(options: {}, (err, responseObj, response))